Bryn Mawr Classical Review.js

```diff
@@ -134,23 +154,13 @@ var testCases = [
 				"itemType": "journalArticle",
 				"creators": [
 					{
-						"firstName": "Christina S.",
-						"lastName": "Kraus",
-						"creatorType": "author"
-					},
-					{
-						"firstName": "Anthony",
-						"lastName": "Grafton",
-						"creatorType": "reviewedAuthor"
-					},
-					{
-						"firstName": "Glenn W.",
-						"lastName": "Most",
-						"creatorType": "reviewedAuthor"
+ 						"firstName": "Keely Elizabeth",
+ 						"lastName": "Heuer",
+ 						"creatorType": "author"
 					},
 					{
-						"firstName": "Salvatore",
-						"lastName": "Settis",
+						"firstName": "Rolf",
+						"lastName": "Hurschmann",
```

Comment:
One article is
http://bmcr.brynmawr.edu/2013/2013-01-04.html
and the other
http://bmcr.brynmawr.edu/2013/2013-01-44.html

@Jmuccigr Just make sure that url you provide and the data are corresponding to each other.